Bottom: 4bf55edbdfd5f0f7f5559c316d0ae5391485c70c
Top:    4b77c49433960f7205d1f8af6f12cb30ba511344
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-02-18 22:45:04 +1300

Refresh of extended-to-stream

---

diff --git a/testtools/testresult/real.py b/testtools/testresult/real.py
index 5b23786..efac704 100644
--- a/testtools/testresult/real.py
+++ b/testtools/testresult/real.py
@@ -1209,7 +1209,6 @@ class ExtendedToStreamDecorator(CopyStreamResult, StreamSummary, TestControl):
 
     def stopTest(self, test):
         self._tags = self._tags.parent
-        pass
 
     @property
     def current_tags(self):
